source/Octopus.Server.Client/Model/SpaceResource.cs

```diff
@@ -20,6 +20,8 @@ public SpaceResource()
         [Writeable]
         public bool IsDefault { get; set; }
 
+        public bool IsPrivate { get; set; }
```

Comment:
I left the `Writeable` attribute off `IsPrivate` in the guess that that's the right move, according to the logic in [this comment](https://github.com/OctopusDeploy/OctopusDeploy/pull/11381/files#r806482691). Is this correct?

Reply:
Yes that would be the right thing to do.